sdk/securitycenter/Microsoft.Azure.Management.SecurityCenter/src/Generated/AssessmentsMetadataOperationsExtensions.cs

```diff
@@ -41,7 +41,7 @@ public static IPage<SecurityAssessmentMetadata> List(this IAssessmentsMetadataOp
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<IPage<SecurityAssessmentMetadata>> ListAsync(this IAssessmentsMetadataOperations operations, CancellationToken cancellationToken = default(CancellationToken))
+            public static async Task<IPage<SecurityAssessmentMetadataResponse>> ListAsync(this IAssessmentsMetadataOperations operations, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
There are quite a few breaking changes like this can you link the swagger spec related to this and did it have the `Approved-BreakingChanges` label?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/15390 - Assessment/AssessmentMetadata
https://github.com/Azure/azure-rest-api-specs/pull/18045 - Alerts

The rest are new.

As for location method change it was created by someone from Azure JS SDK, I'll send you the pr as well:
https://github.com/Azure/azure-rest-api-specs/pull/17958

Sorry it took so long to write a replay, we needed to check about the location changes.

I'll also change version major version (Let me know if minor is enough for this change)